js/apps/kc-create/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "kc-create",
+  "version": "999.0.0-SNAPSHOT",
+  "description": "Create a new Keycloak ui project based on a template",
+  "main": "create.js",
+  "type": "module",
+  "scripts": {
+    "start": "node create.js"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@types/node": "^20.14.2"
+  },
+  "dependencies": {
+    "chalk": "^5.3.0",
+    "commander": "^12.1.0",
+    "fs-extra": "^11.2.0",
+    "mustache": "^4.2.0",
+    "simple-git": "^3.25.0"
+  },
+  "packageManager": "pnpm@9.5.0+sha512.140036830124618d624a2187b50d04289d5a087f326c9edfc0ccd733d76c4f52c3a313d4fc148794a2a9d81553016004e6742e8cf850670268a7387fc220c903"
```

Comment:
This field should only be used in the root `package.json`, otherwise you'll end up with different package manager versions depending on the directory you are in.

```suggestion
```